vermouth/edge_tuning.py

```diff
@@ -336,6 +333,8 @@ def _are_close(threshold, molecules, key_a, key_b):
             return False
         else:
             return length < threshold
+    else:
```

Comment:
Can this be turned around: if a != b: return False and if we're still running do the try-except? Removes a level of indentation.